source/common/grpc/typed_async_client.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
+
+#include <chrono>
+
+#include "envoy/grpc/async_client.h"
+
+namespace Envoy {
+namespace Grpc {
+
+/**
+ * Forward declarations for helper functions.
+ */
+void SendMessageUntyped(RawAsyncStream* stream, const Protobuf::Message& request, bool end_stream);
```

Comment:
Can we do something to indicate that these are internal? It's a bit confusing having untyped methods in a typed async client.

Reply:
Moved into Internal namespace.  PTAL